base/system/chkdsk/lang/ro-RO.rc

```diff
@@ -0,0 +1,38 @@
+/*
+ * PROJECT:     Check Disk utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Romanian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "\n\
+Uz: %s [drive:] [-F] [-V] [-R] [-C] \n\n\
+[drive:]    Specifică modul pilot pentru control.\n\
```

Comment:
I should have looked more carefully at the the English text when I've made my previous comment, and I apologize for that. "Specifies the drive to check" does not have anything to do with a "driver" but a "drive", which is a different concept. First is an intermediary program used to access real or emulated hardware, whereas the second one is a representation for a given information storage unit. Here we may have as well "Specifies the storage unit to check", translatable as "Specifică unitatea de stocare pentru control" (or just "Specifică unitatea de stocare").
I hope you'll still use the term "modul pilot" or "pilot" for "driver" when you'll encounter that term.